DTAdmin/Admins/AdminViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  AdminViewController.swift
+//  DTAdmin
+//
+//  Created by ITA student on 10/15/17.
+//  Copyright © 2017 Yurii Krupa. All rights reserved.
+//
+
+import UIKit
+
+class AdminViewController: UIViewController {
+  
+  @IBOutlet weak var adminsListTableView: UITableView!
+  
+  var adminsList: [UserModel.Admins]?
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    
+    self.title = "Administrators"
+
+    let btnAddAdmin = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(self.showAdminCreateUpdateViewController))
+    let btnSyncData = UIBarButtonItem(barButtonSystemItem: .refresh, target: self, action: #selector(self.syncDataWithServer))
+    self.navigationItem.rightBarButtonItems = [btnAddAdmin, btnSyncData]
+    
+//    MARK: DEBUG - Using for first login into system
+//    _ = NetworkManager().logIn(username: "admin", password: "dtapi_admin") { (admin, cookie) in
+//        print(admin, cookie)
+//        }
+//    MARK: DEBUG - Using to create new user, to preceed should be loginned before
+//    _ = NetworkManager().createAdmin(username: "veselun", password: "1qaz2wsx", email: "veselun@tuhes.if.com")
+
+//    MARK: DEBUG - Using for geting list of admin, to preceed should be loginned before
+//    NetworkManager().getAdmins { (admins) in
+//      print(UserDefaults.standard.getCookie())
+//      print(admins)
+//      self.adminsList = admins
+//      self.adminsListTBV.reloadData()
+//    }
+//    MARK: DEBUG - Using for first login into system, to preceed should be loginned before
+//    _ = NetworkManager().logOut()
+
+  }
+  
+  @objc func showAdminCreateUpdateViewController() {
+    guard let adminCreateUpdateViewController = UIStoryboard(name: "Admin", bundle: nil).instantiateViewController(withIdentifier: "AdminCreateUpdateViewController") as? AdminCreateUpdateViewController else  { return }
+    self.navigationController?.pushViewController(adminCreateUpdateViewController, animated: true)
+  }
+  @objc func syncDataWithServer() {
+//    print("Cookie",UserDefaults.standard.getCookie())
+    NetworkManager().getAdmins { (admins) in
+      self.adminsList = admins
+      self.adminsListTableView.reloadData()
+    }
+  }
+}
+
+private func setUpNavigationBar() {
+}
+
+extension AdminViewController: UITableViewDelegate {
+  
+}
+
+extension AdminViewController: UITableViewDataSource {
+  
+  func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+    return adminsList != nil ? adminsList!.count : 0
+  }
+  
+  func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+    //TODO: make cell more informative
+    let cell = tableView.dequeueReusableCell(withIdentifier: "reusableAdminCell")!
```

Comment:
IMO, `AdminCell` is enough. It's clear that it's reusable identifier

Reply:
`IMO` - how to understand, what this abbreviation means?

Reply:
Which abbreviation? `AdminCell`?
For me it's pretty understandable from the context when you are reading the code
`tableView.dequeueReusableCell(withIdentifier: "AdminCell")`